vsintegration/src/FSharp.Editor/CodeFixes/FSharpCodeFixContext.fs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor
+
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.Text
+
+open CancellableTasks
+
+type FSharpCodeFixContext(document: Document, span: TextSpan) =
+
+    let mutable _sourceText = None
```

Comment:
Underscores are for unused bindings usually

Reply:
Yep, fixed